packages/datepickers/src/styled/StyledDay.ts

```diff
@@ -6,76 +6,106 @@
  */
 
 import styled, { DefaultTheme, ThemeProps, css } from 'styled-components';
-import { retrieveComponentStyles, getColorV8, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
 
 interface IStyledDayProps extends ThemeProps<DefaultTheme> {
-  isPreviousMonth?: boolean;
-  isToday?: boolean;
-  isDisabled?: boolean;
-  isSelected?: boolean;
-  isCompact: boolean;
+  $isPreviousMonth?: boolean;
+  $isToday?: boolean;
+  $isDisabled?: boolean;
+  $isSelected?: boolean;
+  $isCompact: boolean;
 }
 
-const retrieveStyledDayColors = ({
-  isSelected,
-  isDisabled,
-  isToday,
-  isPreviousMonth,
+const sizeStyles = () => {
+  return css`
+    border-radius: 50%;
+    width: 100%;
+    height: 100%;
+  `;
+};
+
+const colorStyles = ({
+  $isSelected,
+  $isDisabled,
+  $isToday,
+  $isPreviousMonth,
   theme
 }: IStyledDayProps) => {
   let backgroundColor = 'inherit';
-  let color = getColorV8('primaryHue', 600, theme);
+  let foreground;
+  const backgroundHover = getColor({
+    variable: 'background.primaryEmphasis',
+    theme,
+    transparency: theme.opacity[100]
+  });
+  const foregroundHover = getColor({
+    variable: 'foreground.primary',
+    light: { offset: 100 },
+    dark: { offset: -100 },
+    theme
+  });
+  const backgroundActive = getColor({
+    variable: 'background.primaryEmphasis',
+    theme,
+    transparency: theme.opacity[200]
+  });
+  const foregroundActive = getColor({
+    variable: 'foreground.primary',
+    light: { offset: 200 },
+    dark: { offset: -200 },
+    theme
+  });
 
-  if (isSelected && !isDisabled) {
-    backgroundColor = getColorV8('primaryHue', 600, theme)!;
-    color = getColorV8('background', 600 /* default shade */, theme);
-  } else if (isDisabled) {
-    color = getColorV8('neutralHue', 400, theme);
-  } else if (isToday) {
-    color = 'inherit';
-  } else if (isPreviousMonth) {
-    color = getColorV8('neutralHue', 600, theme);
+  if ($isSelected && !$isDisabled) {
+    backgroundColor = getColor({ variable: 'background.primaryEmphasis', theme });
+    foreground = getColor({ variable: 'foreground.onEmphasis', theme });
+  } else if ($isDisabled) {
+    foreground = getColor({ variable: 'foreground.disabled', theme });
+  } else if ($isToday) {
+    foreground = 'inherit';
+  } else if ($isPreviousMonth) {
+    foreground = getColor({ variable: 'foreground.subtle', theme });
+  } else {
+    foreground = getColor({ variable: 'foreground.primary', theme });
   }
 
   return css`
     background-color: ${backgroundColor};
-    color: ${color};
+    color: ${foreground};
 
-    ${!isSelected &&
-    !isDisabled &&
-    `
+    ${!$isSelected &&
+    !$isDisabled &&
```

Comment:
Pure CSS is definitely preferred here. We should be able to query against

```
:not([aria-selected]),
:not([aria-disabled]) { ...
```

Reply:
I'm not seeing `aria-selected` used on this component. Are you suggesting I add it / replace `$isSelected` with `aria-selected`? 👀